data/core/units/monsters/Wolf_Dire.cfg

```diff
@@ -16,7 +16,7 @@
     advances_to=null
     cost=33
     usage=scout
-    description=_ "Wolves are aggressive animals that hunt in packs. Although weak individually, a wolf pack can kill even the strongest man in minutes."
+    description=_ "Direwolves are stronger, darker, larger and more dangerous then wolves, making them feared among many travellers. Even their claws can be considered as deadly as a wolf's bite. They typically stand taller than a horse at their shoulder, and have an appetite to match."
```

Comment:
your suggested description says they have claws secondary attack but they dont. The Direwolf Rider does, not the direwolf itself.

Reply:
I removed this line.